src/test/java/nvt/kts/ticketapp/e2e/pages/home/HomePage.java

```diff
@@ -21,6 +21,13 @@
     @FindBy(xpath = "//*[@id=\"paginator\"]/div/div/div[2]/button[2]")
     private WebElement paginatorNextButton;
 
+    @FindBy(className = "mat-select-arrow-wrapper")
+    private WebElement paginatorSelect;
+
+    @FindBy(xpath = "//*[@id=\"event-list\"]/event-component[2]/mat-card/button")
```

Comment:
isto i ovde.. to je najbolje da imas selektor cele liste samo, pa u testu da cupas prvi element ako postoji ako ne nista